### ISSUE TITLE
Enemy reaction timers are hardcoded

### ISSUE THREAD

**Issue by kevinfoley** [eezstreet/SWATEliteForce#145]
AI.ini has a series of properties that are unused:

    [SwatGame.SwatEnemyBase]

    ; These are a new set of timers, to add reaction time to the AI.
    ; These are cumulative with the amount of time necessary to aim
    LowSkillMinTimeBeforeShooting = 5
    LowSkillMaxTimeBeforeShooting = 10
    MediumSkillMinTimeBeforeShooting = 5
    MediumSkillMaxTimeBeforeShooting = 10
    HighSkillMinTimeBeforeShooting = 5
    HighSkillMaxTimeBeforeShooting = 10

However, SwatEnemy.uc does not use the properties from the INI file; instead, they are hardcoded constants:

    const            LowSkillMinTimeBeforeShooting = 0.65;
    const            LowSkillMaxTimeBeforeShooting = 0.9;
    const            MediumSkillMinTimeBeforeShooting = 0.5;
    const            MediumSkillMaxTimeBeforeShooting = 0.65;
    const            HighSkillMinTimeBeforeShooting = 0.3;
    const            HighSkillMaxTimeBeforeShooting = 0.45;

**Comment by kevinfoley** [comment]
Ah, I get a runtime error if I add config vars to SwatEnemy; I assume you ran into the same issue?

**Comment by eezstreet** [comment]
Yeah, unfortunately. I can mark this as `Delayed` though because there is a workaround for it (see how SwatHostage.uc handles the DOA timer and you might get the idea)

**Comment by eezstreet** [comment]
Note to self: PR #150 also has similar issues

**Comment by eezstreet** [comment]
The thinking is that we're going to migrate a lot of config values out of AI.ini and put them into SwatEnemyConfig.ini and SwatHostageConfig.ini as special new config classes that have all the necessary data. This includes other variables which are marked as config, so we can use that space for things that actually *do* get modified on each instance of SwatHostage/SwatEnemy, like DOA timers for instance.
